### PR TITLE
UI-2406 Add customPopup to layer geojson to support html string

### PR DESCRIPTION
**Pull request by hanzge**
~~DO NOT MERGE, IT'S NOT DONE YET.~~

- Add content property which supports html string to popup info/data
- Add customPopup to layer geojson to support html string

### CODE REVIEW

**Comment by mallik-svmx** [comment]
@hanzge 
I am bit confused with this PR implementation. The ask from UI-2406:  "px-map-layer-geojson component does not support to customize its' tooltip for each feature/segment. It just supports to show the tooltip like:<Standard One Screenshot>"

This demo example simulated under - px-map-popup-info-demo.html looks to me not relevant to px-map-layer-geojson. Does it solve the issue of UI-2406?

**Comment by joshsylvester** [comment]
history.md missing

**Comment by leslieli** [comment]
Seem that this is a improvement for px-map-popup-info component, not for px-map-layer-geojson which is UI-2406 for.

**Comment by hanzge** [comment]
@mallik-svmx 

I thought the json format of `px-map-layer-geojson` is like marker group. I'll modify `px-map-layer-geojson` to make it accept custom content.

**Comment by hanzge** [comment]
Merging with 2 approvals. I'll edit release notes right after merging

**Comment by mallik-svmx** [comment]
👍 Just one last step after release notes and version tag publish So, Max Team can start using these features:
- Reference the latest version of px-map and update some demo/examples in ui-components-predix

Thank you, @hanzge @sheela-svmx

File: px-map-behavior-layer-geojson.es6.js [hunk 1]
```diff
@@ -184,6 +184,13 @@
     },
 
     _bindPopup(feature, layer) {
+      const customPopup = feature.customPopup;
+
+      if (customPopup) {
+        const popup = new PxMap.InfoPopup({ ...customPopup });
```
Comment (mallik-svmx) [inline px-map-behavior-layer-geojson.es6.js]: @hanzge 
(I should have caught this before)
With this we are only supporting popup custom content for InfoPop only and not the DataPopup rendered from layerGeoJson, this popup instance should be either InfoPopup Pr DataPopup supporting the configuration from layerGeoJson, this should have been;
const klassName = // feature should define whether InfoPopup/Or DataPopup;
const popup = new PxMap[klassName]({ ...customPopup });

can you please create followup Jira ticket in the backlog, once we clear on this feature gap?

File: px-map-behavior-popup.es6.js [hunk 2]
```diff
@@ -420,8 +430,9 @@
 
       const customContentArr = customContent.map(elem => elem.outerHTML);
       let customContentStr;
-
-      if (customContentArr.length > 1) {
+      if (htmlContent) {
```
Comment (mallik-svmx) [inline px-map-behavior-popup.es6.js]: why is htmlContent only with InfoPopup and not with DataPopup?

**Pull request merged**